modules/ducktests/tests/ignitetest/services/utils/jmx_utils.py

```diff
@@ -76,7 +76,7 @@ def find_mbean(self, pattern, domain='org.apache'):
         :param domain: Domain of MBean
         :return: JmxMBean instance
         """
-        cmd = "echo $'open %s\\n beans -d %s \\n close' | %s | grep -o '%s'" \
+        cmd = "echo $'open %s\\n beans -d %s \\n close' | %s | grep -o '.*%s$'" \
```

Comment:
please explain the changes

Reply:
we can pass a short bean name(ZookeeperDiscoverySpi) instead of 
* group = SPIs, name = ZookeeperDiscoverySpi

Reply:
I found it more convenient. undo the change?

Reply:
returned